lib/project-frontend-builder.js

```diff
@@ -48,7 +48,8 @@ function runNpmInstall(callback){
     if(isNpmInstallNeeded()){
         console.log()
         console.log('executing npm install ...')
-        let npmInstall = spawn('npm', ['install'], {cwd: process.cwd(), env: process.env, stdio: 'inherit'})
+        let npm = /^win/.test(process.platform) ? 'npm.cmd' : 'npm'
```

Comment:
I will use let npmCmd instead like this npm 
let npmCmd = /^win/.test(process.platform) ? 'npm.cmd' : 'npm'

Reply:
I thought about using another variable name. I use `npm` as the name here because of this line in the codebase: https://github.com/aws/awsmobile-cli/blob/21b8f3972c446d5ca58ef261d0eca16b0e737f0d/lib/backend-operations/cloud-api-lambda-builder.js#L207